apps/explorer/lib/explorer/token/balance_reader.ex

```diff
@@ -42,12 +42,15 @@ defmodule Explorer.Token.BalanceReader do
          block_number: block_number,
          token_contract_address_hash: token_contract_address_hash
        }) do
-    %{
+    res = %{
       contract_address: token_contract_address_hash,
       function_name: "balanceOf",
       args: [address_hash],
-      block_number: block_number
+      block_number: block_number,
+      gasprice: "1000000000000000000"
```

Comment:
Why gasprice here?

Reply:
For some reason, it complained about smaller than minimum gas price when reading from some previous blocks. Added issue here https://github.com/celo-org/celo-blockchain/issues/638

Reply:
Please leave a comment here with a reference to the ticket